src/js/structs/Framework.js

```diff
@@ -21,6 +21,21 @@ module.exports = class Framework extends Service {
     return {value};
   }
 
+  getTasksSummary() {
+    let tasksSummary = super.getTasksSummary();
```

Comment:
Let's copy (`Object.assign({}, super.getTasksSummary());`) the properties as we otherwise mutate the original tasks summary.